wordpress-mobile/WordPress-Android
# Add EDITOR_UPLOAD_MEDIA_PAUSED analytics event

When a device is offline and media uploads are ongoing, throw a "paused" analytics event instead of a failure.

Resolves:

* https://github.com/wordpress-mobile/gutenberg-mobile/issues/6516


-----

## To Test:

1. Create a post and upload an image
2. Turn off network connectivity (via airplane mode or similar)
3. Observe that paused event fires instead of failure


-----

## Regression Notes

1. Potential unintended areas of impact
Analytics events

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing in Tracks

3. What automated tests I added (or what prevented me from doing so)
N/A (analytics)

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
